### ISSUE TITLE
How SyzyASAN report more errors than just one？

### ISSUE THREAD

**Issue by kg2012** [google/syzygy#54]
The process terminated by a HEAP-buffer-overflow when i run a instrumented exe，but can i get all of the errors with running the exe once?

**Comment by sebmarchand** [comment]
I'm not sure that I fully understand your problem, what are you trying to do exactly?

**Comment by chhamilton** [comment]
No, you can't detect more than one error, by design. Generally, memory errors can quickly cascade into multiple other memory errors, and the later errors wouldn't have meaningful context. As such, we exit with the first error.
